### PR TITLE
Add aspects to curriculum/javascript/core/scope/

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/scope/hoisting.md?ref=7f0c475e9df16fa335e7d0105f9c3a3833726b9c

File: javascript/core/scope/closures.md [hunk 1]
```diff
@@ -18,6 +18,11 @@ standards:
   javascript.functions.5: 10
 
 
+aspects:
+  - deep
```
Comment (ryanwarsaw) [inline javascript/core/scope/closures.md]: I would remove `deep` and add `introduction` and here's why. This is often one of the first times a programmer is going to deal with the concept of scope, whether they realize it or not. So this is something we want to share with them early on, so they have that exposure.

File: javascript/core/scope/hoisting.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ tags:
 notes: ''
 
 aspects:
-  - introduction
```
Comment (ryanwarsaw) [inline javascript/core/scope/hoisting.md]: I would keep this `introduction`, it's a very common error for beginners.

File: javascript/core/scope/js-practice-closures.md [hunk 3]
```diff
@@ -19,6 +19,10 @@ standards:
   javascript.functions.4: 1000
   javascript.functions.6: 1000
 
+
+aspects:
+  - workout
```
Comment (ryanwarsaw) [inline javascript/core/scope/js-practice-closures.md]: add `introduction`

File: javascript/core/scope/js-practice-hoisting.md [hunk 4]
```diff
@@ -14,6 +14,10 @@ link: http://www.codewars.com/kata/5504b15fd181b977c30001d5/train/javascript
 
 linkType: codewars
 
+
+aspects:
+  - workout
```
Comment (ryanwarsaw) [inline javascript/core/scope/js-practice-hoisting.md]: add `introduction`

**Pull request merged**